cfme/tests/configure/test_access_control.py

```diff
@@ -1097,14 +1099,22 @@ def test_tenant_quota_input_validate(appliance):
 
 
 def test_delete_default_tenant(appliance):
+    """
+    Steps:
+    1. Login as an 'Administrator' user
+    2. Navigate to configuration > access control > tenants
+    3. Select default tenant('My Company') from tenants table
+    4. Delete using 'configuration > Delete selected items'
+    5. Check whether default tenant is deleted or not
+    """
     roottenant = appliance.collections.tenants.get_root_tenant()
-    view = navigate_to(appliance.collections.tenants, 'All')
-    for row in view.table.rows():
-        if row.name.text == roottenant.name:
-            row[0].check()
-    msg = 'Default Tenant "{}" can not be deleted'.format(roottenant.name)
-    with pytest.raises(Exception, match=msg):
-        view.toolbar.configuration.item_select('Delete selected items', handle_alert=True)
+    try:
+        msg = 'Default Tenant "{}" can not be deleted'.format(roottenant.name)
+        tenant = appliance.collections.tenants.instantiate(name=roottenant.name)
+        tenant_obj = appliance.collections.tenants
```

Comment:
Required:
we can go with a good name like `tenant_collection`. `appliance.collections.tenants` not representing `tenant object` but its `collection`.
or
you can simply go for deletion like `appliance.collections.tenants.delete(tenant)`

Reply:
I am going with simple approach ```appliance.collections.tenants.delete(tenant)```. which will remove extra variable.
Thank you